marklogic-data-hub/src/main/resources/hub-internal-config/security/privileges/data-hub-read-step-definition.json

```diff
@@ -0,0 +1,5 @@
+{
+  "privilege-name": "data-hub:readStepDefinition",
```

Comment:
What's this needed for? Does it map to something in HC?

Reply:
@rjrudin The privilege/authority is for custom steps. I can change it to readCustom if we think it makes more sense. It is listed as stepDefinition in the roles wiki and is similar to readIngestion being closer to what we have on the backed. Perhaps we should use readCustom because we'll create separate custom steps aside from step definitions in the future?